libraries/mediaplayer/impl/src/main/kotlin/io/element/android/libraries/mediaplayer/impl/MediaPlayerImpl.kt

```diff
@@ -130,7 +145,10 @@ class MediaPlayerImpl @Inject constructor(
             if (!_state.value.isPlaying) return
             delay(100)
             _state.update {
-                it.copy(currentPosition = player.currentPosition)
+                it.copy(
+                    currentPosition = player.currentPosition,
+                    duration = player.duration,
```

Comment:
Is this really needed? Once we update the duration whenever the media has changed that should be enough shouldn't it? (i mean... I think once loaded a media can't mutate its duration)